internal/connector/internal/workers/cluster_mgr.go

```diff
@@ -93,10 +93,10 @@ func (m *ClusterManager) doReconcile(errs *[]error, kind string,
 		}
 
 		if len(serrs) != 0 {
-			return errors.GeneralError("Error reconciling %s clusters: %s", kind, serrs)
+			return errors.GeneralError("Error reconciling %s clusters: %v", kind, serrs)
```

Comment:
Error messages should start with lowercase letter

Reply:
fixed